client/src/modules/accounts/edit/accounts.edit.modal.html

```diff
@@ -56,18 +56,20 @@
 
     <!-- TODO(@jniles): can this just use ng-options? -->
     <div class="form-group"
-         ng-class="{'has-error' : (AccountForm.type_id.$invalid && AccountForm.$submitted) || AccountEditCtrl.invalidTitleAccount}">
+         ng-class="{'has-error' : (AccountForm.type_id.$invalid && AccountForm.$submitted) || AccountEditCtrl.invalidTitleAccount}">         
       <label class="control-label" translate>FORM.LABELS.ACCOUNT_TYPE</label>
       <select
         class="form-control"
         ng-model="AccountEditCtrl.account.type_id"
         ng-change="AccountEditCtrl.titleChangedValidation(AccountEditCtrl.account.type_id)"
         name="type_id"
+        ng-disabled="!AccountEditCtrl.isCreateState"
```

Comment:
To make this less confusing, it would be better to use `ngIf` on the whole form group and then uncomment lines 50 - 55 of this file.